assets/app/scripts/app.js

```diff
@@ -158,7 +158,8 @@ angular
         templateUrl: 'views/pods.html'
       })
       .when('/project/:project/browse/pods/:pod', {
-        templateUrl: 'views/browse/pod.html'
+        templateUrl: 'views/browse/pod.html',
+        reloadOnSearch: false
       })
```

Comment:
I'd like to see us declare our controller here like this:

```javascript
.when('/project/:project/browse/pods/:pod', {
  templateUrl: 'views/browse/pod.html',
  controller: 'PodController',
  reloadOnSearch: false
});
```
rather than with the `ng-controller="PodController"` directive.  If we can then stick to one controller on the page, it will help us push more work into services and directives.  This is probably a great time to try this because we will have multiple things going on (details, metrics, logs, events) and will have to work hard to keep our controller clean.